### PR TITLE
NAS-121171 / 23.10 / Using the new `ErrorHandlerService`

### PR DESCRIPTION
**Pull request by RehanY147**

### CODE REVIEW

File: src/app/modules/entity/utils.ts [hunk 1]
```diff
@@ -24,152 +17,156 @@ export interface FlattenedData extends Record<string, unknown> {
 /**
  * TODO: Likely outdated.
  */
-interface HttpError {
-  code: number;
-  error: {
-    [field: string]: string[];
-  };
-}
+// interface HttpError {
```
Comment (dszidi) [inline src/app/modules/entity/utils.ts:20]: We should delete these if not in use

**Review by dszidi — CHANGES_REQUESTED** [review]
The code looks reasonable. The only thing I would change is
- ditch the commented out EntityUtils code.
- Add some spec files to test make sure the ErrorHandlerService produces desired behavior.

**Comment by RehanY147** [comment]
Tests will be added by https://ixsystems.atlassian.net/browse/NAS-121305

**Review by dszidi — APPROVED** [review]
Tested by triggering the dialog after init on snapshots list page. 
LGTM

image

**Pull request merged**